step_definitions/ui/offer_details_steps.js

```diff
@@ -10,15 +10,10 @@ module.exports = function offer_details_steps() {
         merchantName    = 'DisneyStore.com',
         offerTitle      = 'Free shipping on $75+';
 
-    this.Then(/^an Affiliate Offer is in Shop$/, function(callback) {
-
-        this.allOffersPage = this.homePage.goToOffersPage();
-
-        callback();
-    });
-
     this.Then(/^I ask for offer details$/, function(done) {
 
+        this.allOffersPage = this.allOffersPage = this.homePage.goToOffersPage();
```

Comment:
I think you just want `this.allOffersPage = this.homePage.goToOffersPage();`, right?